frontend/src/firebase/StaffAuth.js

```diff
@@ -1,4 +1,3 @@
-import React, { useState } from "react";
```

Comment:
I think we only needed to remove the "React, " from the import, not the entire line